libraries/rush-lib/src/logic/base/BaseShrinkwrapFile.ts

```diff
@@ -34,7 +34,11 @@ export abstract class BaseShrinkwrapFile {
    *
    * @virtual
    */
-  public validateShrinkwrapAfterUpdate(rushConfiguration: RushConfiguration, terminal: ITerminal): void {}
+  public validateShrinkwrapAfterUpdate(
+    rushConfiguration: RushConfiguration,
+    subspace: Subspace,
```

Comment:
Should this be


```suggestion
    subspace: Subspace | undefined,
```

?

Reply:
I'm guessing not. The future is that Rush Subspaces will be an always-on feature, not an opt-in feature.  The opt-in part is adding more than one subspace beyond the default one.  Thus the APIs generally should NOT accept `undefined`